### ISSUE TITLE
[Bug]: sd-webui-controlnet doesn't work 【It can't work normally under linux, it has been tested in colab】

### ISSUE THREAD

**Issue by triggered96** [Mikubill/sd-webui-controlnet#714]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

I've tried sd-webui-controlnet really hard, but it doesn't work. After nearly a week of research, I still can't recognize the skeleton, outline, etc. The result of the Preview annotator is either all white or all black.

I'm currently on the verge of a nervous breakdown. hope based on help

### Steps to reproduce the problem

![1](https://user-images.githubusercontent.com/117959077/230453056-4645eb67-fda7-47e4-bfca-f949b9658a1a.jpg)
Note: I accidentally turned off the Enable above when I took a screenshot, but it was actually turned on

![2](https://user-images.githubusercontent.com/117959077/230455858-fd7d14e7-5268-471d-ad55-fc0223ebc04b.jpg)

![3](https://user-images.githubusercontent.com/117959077/230455955-4881088e-5ce7-4e0b-a644-5bc5285a5a10.jpg)

![4](https://user-images.githubusercontent.com/117959077/230456036-8b3b4f01-8e28-494a-a795-e4a9e0b107fc.jpg)

### What should have happened?

I want to get a line drawing, or a human skeleton. Instead of being completely blank, or completely black. as shown in the picture above

### Commit where the problem happens

webui: 
controlnet: 

I don't seem to know how to do this

### Command Line Arguments

```Shell
I don't seem to know how to do this
```

### Console logs

```Shell
Loading model: control_scribble-fp16 [c508311e]
Loaded state_dict from [/home/ubuntu/1/stable-diffusion-webui/extensions/sd-webui-controlnet/models/control_scribble-fp16.safetensors]
ControlNet model control_scribble-fp16 [c508311e] loaded.
using mask as input
Loading preprocessor: scribble
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [00:49<00:00,  2.49s/it]
Total progress: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [00:49<00:00,  2.49s/it]
Closing server running on port: 7860

```
```


### Additional information

_No response_

**Comment by triggered96** [comment]
COMMANDLINE_ARGS="--listen --xformers --enable-insecure-extension-access --theme dark" ./webui.sh

**Comment by ljleb** [comment]
The preprocessor is doing exactly what you told it do do. You do not need a preprocessor when passing a scribble directly, you need a preprocessor to derive a scribble from a real picture.

**Comment by ljleb** [comment]
Sorry I am a bit confused because there seems to be 2 different issues at play here.

For the first image, the preprocessor is doing exactly what you told it do do. You do not need a preprocessor when passing a scribble directly, you need a preprocessor to derive a scribble from a real picture.

For the second image, you may be encountering a problem with the preprocessor. Any logs when using the openpose preprocessor? (IIUC only the logs for scribbles have been included above)

**Comment by triggered96** [comment]
> 

```
Applying cross attention optimization (Doggettx).
Textual inversion embeddings loaded(1): EasyNegative
Model loaded in 7.2s (create model: 0.9s, apply weights to model: 2.2s, apply half(): 1.9s, load VAE: 1.5s, move model to device: 0.6s).
*Deforum ControlNet support: enabled*
Running on local URL:  http://0.0.0.0:7860

To create a public link, set `share=True` in `launch()`.
Loading model: control_canny-fp16 [e3fe7712]
Loaded state_dict from [/home/ubuntu/1/stable-diffusion-webui/extensions/sd-webui-controlnet/models/control_canny-fp16.safetensors]
ControlNet model control_canny-fp16 [e3fe7712] loaded.
using mask as input
Loading preprocessor: canny
  0%|                                                                                                                                                                                                        | 0/20 [00:00<?, ?it/s]Warning: StyleAdapter and cfg/guess mode may not works due to non-batch-cond inference
 60%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████▌                                                                            | 12/20 [00:21<00:14,  1.77s/it]
Total progress:  60%|█████████████████████████████████████████████████████████████████████████████████████████████████████████                                                                      | 12/20 [00:19<00:14,  1.76s/it]
```

**Comment by triggered96** [comment]
> Sorry I am a bit confused because there seems to be 2 different issues at play here.
> 
> For the first image, the preprocessor is doing exactly what you told it do do. You do not need a preprocessor when passing a scribble directly, you need a preprocessor to derive a scribble from a real picture.
> 
> For the second image, you may be encountering a problem with the preprocessor. Any logs when using the openpose preprocessor? (IIUC only the logs for scribbles have been included above)

```
Loading weights [fc2511737a] from /home/ubuntu/1/stable-diffusion-webui/models/Stable-diffusion/chilloutmix_NiPrunedFp32Fix.safetensors
Creating model from config: /home/ubuntu/1/stable-diffusion-webui/configs/v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Applying cross attention optimization (Doggettx).
Textual inversion embeddings loaded(1): EasyNegative
Model loaded in 7.2s (create model: 0.9s, apply weights to model: 2.2s, apply half(): 1.9s, load VAE: 1.5s, move model to device: 0.6s).
*Deforum ControlNet support: enabled*
Running on local URL:  http://0.0.0.0:7860

To create a public link, set `share=True` in `launch()`.
Loading model: control_canny-fp16 [e3fe7712]
Loaded state_dict from [/home/ubuntu/1/stable-diffusion-webui/extensions/sd-webui-controlnet/models/control_canny-fp16.safetensors]
ControlNet model control_canny-fp16 [e3fe7712] loaded.
using mask as input
Loading preprocessor: canny
  0%|                                                                                                                                                                                                        | 0/20 [00:00<?, ?it/s]Warning: StyleAdapter and cfg/guess mode may not works due to non-batch-cond inference
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [00:36<00:00,  1.80s/it]
Total progress: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [00:34<00:00,  1.75s/it]
```

**Comment by triggered96** [comment]
![1](https://user-images.githubusercontent.com/117959077/230535741-c66dcda6-be42-40ad-b913-430b676b8e9c.jpg)
![2](https://user-images.githubusercontent.com/117959077/230535745-cb826a7a-10b3-4c4f-b060-fe53adf26a83.jpg)

```
Launching Web UI with arguments: --listen --disable-safe-unpickle --no-half-vae --xformers --enable-insecure-extension-access --medvram --theme dark
[AddNet] Updating model hashes...
0it [00:00, ?it/s]
[AddNet] Updating model hashes...
0it [00:00, ?it/s]
Loading weights [fc2511737a] from /home/ubuntu/1/stable-diffusion-webui/models/Stable-diffusion/chilloutmix_NiPrunedFp32Fix.safetensors
Creating model from config: /home/ubuntu/1/stable-diffusion-webui/configs/v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Applying cross attention optimization (Doggettx).
Textual inversion embeddings loaded(1): EasyNegative
Model loaded in 5.8s (create model: 0.9s, apply weights to model: 1.8s, apply half(): 1.5s, load VAE: 1.5s).
*Deforum ControlNet support: enabled*
Running on local URL:  http://0.0.0.0:7860

To create a public link, set `share=True` in `launch()`.
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [00:26<00:00,  1.34s/it]
Total progress: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [00:27<00:00,  1.36s/it]
Loading model: control_openpose-fp16 [9ca67cc5]
Loaded state_dict from [/home/ubuntu/1/stable-diffusion-webui/extensions/sd-webui-controlnet/models/control_openpose-fp16.safetensors]
ControlNet model control_openpose-fp16 [9ca67cc5] loaded.
using mask as input
Loading preprocessor: openpose
  0%|                                                                                                                                                                                                        | 0/20 [00:00<?, ?it/s]Warning: StyleAdapter and cfg/guess mode may not works due to non-batch-cond inference
 40%|████████████████████████████████████████████████████████████████████████████▊                                                                                                                   | 8/20 [00:14<00:22,  1.84s/it]
Total progress:  40%|██████████████████████████████████████████████████████████████████████▍                                                                                                         | 8/20 [00:15<00:22,  1.91s/it]
Loading model from cache: control_openpose-fp16 [9ca67cc5]
using mask as input
Loading preprocessor: openpose
  0%|                                                                                                                                                                                                        | 0/20 [00:00<?, ?it/s]Warning: StyleAdapter and cfg/guess mode may not works due to non-batch-cond inference
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [00:35<00:00,  1.78s/it]
Total progress: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [00:35<00:00,  1.79s/it]
Closing server running on port: 7860
Restarting UI...
[AddNet] Updating model hashes...
0it [00:00, ?it/s]
Running on local URL:  http://0.0.0.0:7860

To create a public link, set `share=True` in `launch()`.
Loading model: control_openpose-fp16 [9ca67cc5]
Loaded state_dict from [/home/ubuntu/1/stable-diffusion-webui/extensions/sd-webui-controlnet/models/control_openpose-fp16.safetensors]
ControlNet model control_openpose-fp16 [9ca67cc5] loaded.
using mask as input
Loading preprocessor: openpose
  0%|                                                                                                                                                                                                        | 0/20 [00:00<?, ?it/s]Warning: StyleAdapter and cfg/guess mode may not works due to non-batch-cond inference
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [02:00<00:00,  6.03s/it]
Total progress: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [02:00<00:00,  6.03s/it]
Loading model from cache: control_openpose-fp16 [9ca67cc5]
using mask as input
Loading preprocessor: openpose
  0%|                                                                                                                                                                                                        | 0/20 [00:00<?, ?it/s]Warning: StyleAdapter and cfg/guess mode may not works due to non-batch-cond inference
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [01:10<00:00,  3.51s/it]
Total progress: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [01:10<00:00,  3.53s/it]

```

**Comment by triggered96** [comment]
I still can't get it to work, my mental state is really bad right now.

**Comment by triggered96** [comment]
I tried colab, still the same, pure black. `https://colab.research.google.com/github/camenduru/stable-diffusion-webui-colab/blob/main/nightly/chillout_mix_webui_colab.ipynb`

https://github.com/camenduru/stable-diffusion-webui-colab

**Comment by triggered96** [comment]
I wonder if it is under linux that the pictures uploaded by pictures cannot be identified? I am in colab, and the camenduru/stable-diffusion-webui-colab project is still very popular, and I found that the same problem exists. You can reproduce this problem in colab.

**Comment by triggered96** [comment]
I think it should be a bug, because I see that many Windows users have not encountered this problem, but this problem will occur under Linux.

**Comment by zivky** [comment]
same problem here, have you found the solution perhaps?
